pyproject.toml

```diff
@@ -20,7 +20,8 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.7"
 cryptography = ">=2.9.2"
-zeroconf = ">=0.31.0"
+zeroconf = ">=0.32.0"
+asynctest = "^0.13.0"
```

Comment:
I'm guessing `asynctest` should be in the dev-dependencies section? (`poetry add -D asynctest`).

Reply:
Finished testing on 3 production installs.  

I'll push a commit to move it in a second

Reply:
@Jc2k Should be good to go for 0.4.0 👍